templates-extension.md

```diff
@@ -1,341 +1,115 @@
-# Source Templates Extension
+# GEDCOM Citation Source Templates Extension
 
-## Use of Property Sets and Templates
+## Purpose
 
-[Citations: Goals and Considerations (fhiso.org)](https://fhiso.org/TR/citation-goals) says:
+Many types of sources exist with a common set of elements that can be used to format a citation.
+In order to facilitate formatting of consistent citations, it is desirable to allow
+specifying such information with a master source.
 
-> Property set design requires a trade-off between complexity and expressiveness. Academic
```

Comment:
I'd rather keep the requirements, motivation, and survey of existing solutions in the same file rather than splitting into two files.